python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/slice_conditions.py

```diff
@@ -87,6 +87,18 @@ def compute_slice(self, context: SchedulingContext) -> AssetSlice:
             return context.asset_graph_view.create_empty_slice(context.asset_key)
 
 
+@whitelist_for_serdes
+class NewlyRequestedCondition(SliceSchedulingCondition):
```

Comment:
Should this just be `RequestedOnPreviousTick`. Seems more obvious?

Reply:
Let's go with "Newly" terminology now. I still think we might come up with something bettetr in the next few weeks but this is concise and standardized and it is a "two-way door" for now